Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if os(Linux) || os(FreeBSD) || os(iOS)
```

Comment:
I think it's better assume that SwiftFoundation is default. So I suggest introducing flag with name like USE_OBJC_FOUNDATION.